pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
```

Comment:
If we are displaying this title to the users then it must be localizable. If you want to keep a `unique_together`, something that makes sense because of all the formats that share the .properties extension, then I suggest adding an internal name that can be used for the unique check, like:

```python
POOTLE_FORMATS = [
    ("po",
     dict(title=_('Gettext PO'),
          name='Gettext PO'
          extension="po",
          template_extension="pot")),
```